hs_core/templates/pages/baseresource.html

```diff
@@ -850,7 +850,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                                 data-placement="auto"
                                                 {% endif %}
                                                 data-bag-url="{{ cm.bag_url }}">
-                                                <span class="fa fa-briefcase icon-blue"></span>
+                                                <span class="fa fa-cloud-download icon-blue"></span>
```

Comment:
after you click this span, the icon gets resized.
it should remain the same size

https://github.com/user-attachments/assets/e05c964a-85cf-4360-8d95-043dafd6663e

Reply:
Updated the code